app/client/src/components/pages/Community.Tabs.Monitoring.js

```diff
@@ -917,8 +924,7 @@ function MonitoringTab({ monitoringDisplayed, setMonitoringDisplayed }) {
                       <span>All Monitoring Locations</span>
                     </div>
                   </th>
-                  <th>Location Count</th>
-                  <th>Sample Count</th>
+                  <th colspan="2">Location Count</th>
```

Comment:
```suggestion
                  <th>Location Count</th>
```

I don't think the `colspan` is necessary here.